{"uid":"619d201a68ffdba8","name":"Валидация: создание борды с именем — ожидаемый статус 400","fullName":"tests.test_backend.board.board_validation_test#test_create_board_name_validation","historyId":"39b670d8fc35f10e0590532ae0411b51","time":{"start":1749743620419,"stop":1749743621031,"duration":612},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":true,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"owner_client","time":{"start":1749743617438,"stop":1749743617914,"duration":476},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"base_url","time":{"start":1749743617438,"stop":1749743617438,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"_verify_url","time":{"start":1749743617438,"stop":1749743617438,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"delete_output_dir","time":{"start":1749743617438,"stop":1749743617438,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"pytestconfig","time":{"start":1749743617438,"stop":1749743617438,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_space","time":{"start":1749743617914,"stop":1749743618824,"duration":910},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_project","time":{"start":1749743618824,"stop":1749743620418,"duration":1594},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"passed","steps":[{"name":"Проверка, что API вернул статус 400 при empty string","time":{"start":1749743621031,"stop":1749743621031,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"attachments":[{"uid":"869b0182c5d4323a","name":"stdout","source":"869b0182c5d4323a.txt","type":"text/plain","size":79}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":1,"hasContent":true},"afterStages":[{"name":"temp_space::0","time":{"start":1749743651661,"stop":1749743652253,"duration":592},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"backend"},{"name":"parentSuite","value":"tests.test_backend.board"},{"name":"suite","value":"board_validation_test"},{"name":"host","value":"pkrvmxyh4eaekms"},{"name":"thread","value":"2521-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_backend.board.board_validation_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"expected_status","value":"400"},{"name":"name","value":"''"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":6,"skipped":0,"passed":1,"unknown":0,"total":7},"items":[{"uid":"75efff2a2200e8b7","reportUrl":"https://vaizcom.github.io/autotests/119//#testresult/75efff2a2200e8b7","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fc7b5913490>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742684605,"stop":1749742684605,"duration":0}},{"uid":"61863e21ea282472","reportUrl":"https://vaizcom.github.io/autotests/118//#testresult/61863e21ea282472","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9cb88f1f10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742532389,"stop":1749742532389,"duration":0}},{"uid":"26023b4768aa3684","reportUrl":"https://vaizcom.github.io/autotests/117//#testresult/26023b4768aa3684","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f2b9f21f250>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742363881,"stop":1749742363881,"duration":0}},{"uid":"14e4cc125ca23a63","reportUrl":"https://vaizcom.github.io/autotests/116//#testresult/14e4cc125ca23a63","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7efcc7f78ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749741097945,"stop":1749741097945,"duration":0}},{"uid":"11e3723cf5ce2af4","reportUrl":"https://vaizcom.github.io/autotests/115//#testresult/11e3723cf5ce2af4","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f28d1ddcd10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740692501,"stop":1749740692501,"duration":0}},{"uid":"91c5dbe566370c0","reportUrl":"https://vaizcom.github.io/autotests/114//#testresult/91c5dbe566370c0","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f271b110ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740515963,"stop":1749740515963,"duration":0}}]},"tags":["backend"]},"source":"619d201a68ffdba8.json","parameterValues":["400","''"]}